bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/configuration/XmlChannelConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.xmltv.internal.configuration;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link XmlChannelConfiguration} class contains fields mapping
+ * Channel thing configuration parameters.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public class XmlChannelConfiguration {
+    public static final String CHANNEL_ID = "channelId";
+
+    @NonNullByDefault({})
+    public String channelId;
+    @NonNullByDefault({})
+    public Integer offset;
+    @NonNullByDefault({})
```

Comment:
Please place null annotations between the modifiers and type. See also the [Null Annotations Conventions](https://www.openhab.org/docs/developer/development/conventions.html#null-annotations).